package.json

```diff
@@ -19,9 +16,5 @@
       "pre-commit": "npm test"
     }
   },
-  "repository": "tldr-pages/tldr",
```

Comment:
I am not sure why these are not required. Seemed helpful to me.

Reply:
What is its use? If someone needs any metadata, it's all available in tldr-node-client's package.json.

Reply:
This metadata is for this repo. I agree that the name clashes with the node client. In that case, I think we can change it to tldr-pages, and remove the version. And keep everything else. Even if there are no downstream consumers of this package, keeping this metadata doesn't do any harm.

I would prefer to keep it. That said, if you feel strongly about it, and others agree, I won't be stopping you :).

Reply:
We have all that information already stored in node client's [`package.json`](https://github.com/tldr-pages/tldr-node-client/blob/master/package.json). It's also publicly available from there, so we're just keeping duplicate metadata. That was my main motivation.

Reply:
I agree with @agnivade here. It's not harmful to keep the information.

Reply:
It's not harmful if the information is correct. However, here at least 3 tags have incorrect information: `description`, `version`, `license`.

Reply:
True, so let's update them instead of deleting them?

Reply:
I'm for deleting `version` in any case, unless we are going to officially publish the pages on npm.

Reply:
Agree with @sbrl let's fix what's wrong and delete what cannot be fixed. I agree that the `version` field does not make sense in our case, but the others can stay.